src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,107 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-color;
+}
+
+.product-card {
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__product-image {
+    height: 134px;
+    margin: 16px 0 40px;
+    padding-inline: 3px;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__product-title {
+    text-align: left;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__product-code {
+    margin: 4px 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-color;
+  }
+
+  &__product-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews-text {
+    margin: 0;
+    font-size: 10px;
+    max-height: 10px;
+  }
+
+  &__product-price {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__number {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__buy-button {
+    width: 100%;
+    height: 40px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: none;
+    padding: 0;
+
+    color: white;
+    font-family: inherit;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      width: 100%;
+      height: 40px;
+      background-color: white;
+      border-radius: 5px;
+      border: 1px solid #00acdc;
+      padding: 0;
+
+      color: #00acdc;
+      font-family: inherit;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+    }
+  }
+}
```

Comment:
The `product-stars` block is missing from this SCSS file. Ensure that you define styles for the `product-stars` block as it is used in the HTML file. This will help maintain consistency and ensure that all elements are styled correctly.